config/environment.rb

```diff
@@ -3,3 +3,14 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+# Set ActionMailer to use Sendgrid - CHANGE TO PRODUCTION SETTINGS UPON DEPLOYMENT
+ActionMailer::Base.smtp_settings = {
+  user_name: 'patmbolger',
+  password:  '38LjEPStn-',
+  domain:    'herokuapp.com',
+  address:   'smtp.sendgrid.net',
+  port:       587,
+  authentication: :plain,
+  enable_starttls_auto: true  
+}
```

Comment:
I will use the 'figaro' gem - it has direct support for Heroku deployment (e.g. "figaro heroku:set").